src/app/[locale]/types/typescript.ts

```diff
@@ -0,0 +1,61 @@
+// Define a type for the task data
+export type TaskData = {
+  _id?: string;
+  title: string;
+  description: string;
+  type: string;
+  createdOn: string;
+  status: string;
+  assignedTo?: any;
+  group?: any;
+};
+
+// export type Context = {
```

Comment:
if its not used it should be removed from PR

Reply:
removed